### PR TITLE
docs: add installation guide

### PR DESCRIPTION
**Pull request by kkujawa-sumo**

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -1 +1,83 @@
 # Installation
+
+Sumo Logic Open Telemetry distribution can be run either using binary file available in [Github releases][github_releases] or
+using container images which are stored in AWS Public ECR under the following repository:
+`public.ecr.aws/sumologic/sumologic-otel-collector`.
+
+## Standalone
+
+Sumo Logic Open Telemetry distribution is a static Go binary. To run it standalone you need to only run binary file
+downloaded from [Github releases][github_releases] with appropriate configuration.
```
Comment (pmalek-sumo) [inline docs/Installation.md:10]:
```suggestion
Sumo Logic Open Telemetry distribution is a static Go binary.
To run it as standalone process you only need to run the binary file downloaded from
[Github releases][github_releases] with appropriate configuration.
```

File: docs/Installation.md [hunk 2]
```diff
@@ -1 +1,83 @@
...
+    sudo mv otelcol-sumo /usr/local/bin/otelcol-sumo
+    ```
+
+1. Verify installation:
+
+    ```bash
+    otelcol-sumo --version
+    ```
+
+1. Prepare configuration according [this](Configuration.md) documentation and save it in `config.yaml`
```
Comment (pmalek-sumo) [inline docs/Installation.md:43]:
```suggestion
1. Prepare configuration according to [this](Configuration.md) documentation and save it in `config.yaml`
```

File: docs/Installation.md [hunk 3]
```diff
@@ -1 +1,83 @@
...
+1. Run Sumo Logic Open Telemetry distribution:
+
+   ```bash
+   otelcol-sumo --config config.yaml
+   ```
+
+## Container image
+
+In order to run Sumo Logic Open Telemetry distribution in container you need to only run container image available in
+`public.ecr.aws/sumologic/sumologic-otel-collector` repository.
```
Comment (pmalek-sumo) [inline docs/Installation.md:54]:
```suggestion
In order to run Sumo Logic Open Telemetry distribution in a container you only need to run the container
using the image available in `public.ecr.aws/sumologic/sumologic-otel-collector` repository.
```

File: docs/Installation.md [hunk 4]
```diff
@@ -1 +1,83 @@
...
+
+1. Set the release version variable:
+
+   ```bash
+   export RELEASE_VERSION=0.0.10
+   ```
+
+1. Prepare configuration according [this](Configuration.md) documentation and save it in `config.yaml`
```
Comment (pmalek-sumo) [inline docs/Installation.md:62]:
```suggestion
1. Prepare configuration according to [this](Configuration.md) documentation and save it in `config.yaml`
```

File: docs/Installation.md (hunk 1 above)
Comment (kkujawa-sumo) [reply docs/Installation.md]: Corrected.

File: docs/Installation.md (hunk 2 above)
Comment (kkujawa-sumo) [reply docs/Installation.md]: Corrected.

File: docs/Installation.md (hunk 3 above)
Comment (kkujawa-sumo) [reply docs/Installation.md]: Corrected.

File: docs/Installation.md (hunk 4 above)
Comment (kkujawa-sumo) [reply docs/Installation.md]: Corrected.

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**